external/avro/src/main/scala/org/apache/spark/sql/avro/SparkAvroDatumWriter.scala

```diff
@@ -125,42 +125,42 @@ class SparkAvroDatumWriter[D](rootAvroType: Schema, rootCatalystType: DataType,
       catalystType: DataType, avroType: Schema, path: List[String]
   ): Writer = {
     val baseWriter: Writer = (catalystType, avroType.getType) match {
-      case (NullType, NULL) => getNullWriter(avroType)
-      case (BooleanType, BOOLEAN) => getBooleanWriter(avroType)
-      case (ByteType, INT) => getByteAsIntWriter(avroType)
-      case (ShortType, INT) => getShortAsIntWriter(avroType)
-      case (IntegerType, INT) => getIntWriter(avroType)
-      case (LongType, LONG) => getLongWriter(avroType)
-      case (FloatType, FLOAT) => getFloatWriter(avroType)
-      case (DoubleType, DOUBLE) => getDoubleWriter(avroType)
+      case (NullType, NULL) => makeNullWriter(avroType)
+      case (BooleanType, BOOLEAN) => makeBooleanWriter(avroType)
+      case (ByteType, INT) => makeByteAsIntWriter(avroType)
+      case (ShortType, INT) => makeShortAsIntWriter(avroType)
+      case (IntegerType, INT) => makeIntWriter(avroType)
+      case (LongType, LONG) => makeLongWriter(avroType)
+      case (FloatType, FLOAT) => makeFloatWriter(avroType)
+      case (DoubleType, DOUBLE) => makeDoubleWriter(avroType)
```

Comment:
Changing names of methods from getXYZWriter to makeXYZWriter since that better reflects the methods.